Source/r_main.c

```diff
@@ -620,7 +620,7 @@ void R_RenderPlayerView (player_t* player)
     { // killough 2/10/98: add flashing red HOM indicators
       byte c[47*47];
       extern int lastshottic;
-      int i , color = !flashing_hom || (gametic % 20) < 9 ? 0xb0 : 0;
+      int i , color = !flashing_hom || (gametic % 20) < 9 ? 0 : 0xb0;
```

Comment:
Why did you change the swapping sequence?

Reply:
Because by default I want to fill with black color, not red. Not sure I understand this code.

Reply:
Ah, okay. So, `autodetect_hom` is the setting that can only be enabled by a cheat code whereas `flashing_hom` can be enabled by the menu.

Reply:
Could you please add a comment why you chose to change the colors?